tests/arp/test_arpall.py

```diff
@@ -0,0 +1,159 @@
+import logging
+import pytest
+import time
+from datetime import datetime
+
+from ptf_runner import ptf_runner
+from common.helpers.assertions import pytest_assert
+
+@pytest.fixture(scope="module")
+def common_setup_teardown(duthost, ptfhost):
+    mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
+    int_facts = duthost.interface_facts()['ansible_facts']
+
+    ports = list(sorted(mg_facts['minigraph_ports'].keys(), key=lambda item: int(item.replace('Ethernet', ''))))
+
+    # Select port index 0 & 1 two interfaces for testing
+    intf1 = ports[0]
+    intf2 = ports[1]
+    logging.info("Selected ints are {0} and {1}".format(intf1, intf2))
+
+    intf1_indice = mg_facts['minigraph_port_indices'][intf1]
+    intf2_indice = mg_facts['minigraph_port_indices'][intf2]
+
+    po1 = get_po(mg_facts, intf1)
+    po2 = get_po(mg_facts, intf2)
+    
+    try:
+        # Make sure selected interfaces are not in portchannel
+        if po1 is not None:
+            duthost.shell('config portchannel member del {0} {1}'.format(po1, intf1))
+            duthost.shell('config interface startup {0}'.format(intf1))
+        
+        if po2 is not None:
+            duthost.shell('config portchannel member del {0} {1}'.format(po2, intf2))
+            duthost.shell('config interface startup {0}'.format(intf2))
+        
+        # Change SONiC DUT interface IP to test IP address
+        duthost.shell('config interface ip add {0} 10.10.1.2/28'.format(intf1))
+        duthost.shell('config interface ip add {0} 10.10.1.20/28'.format(intf2))
+
+        if (po1 is not None) or (po2 is not None):
+            time.sleep(40)
```

Comment:
I want to raise another PR to create a wait_until function for replacing the hardcode sleep. Because test_po_update.py has the same issue.